docs/data-cookbooks/throttling-queued-executions.md

```diff
@@ -16,29 +16,70 @@ Limiting the number of executions that can be running from a given queue is usef
+
+Define the `throttled_queues` variable for the `cumulus` module in your [Cumulus deployment](./../deployment/README.md#deploy-the-cumulus-instance) to specify the maximum concurrent executions for the queue.
+
+> **Please note:** The `id` used to identify the new queue is arbitrary, but **will be used to refer to this queue later when configuring workflows and rules**.
+
+```hcl
+module "cumulus" {
+  # ... other variables
+
+  throttled_queues = [{
+    id = "backgroundJobQueue",
+    url = aws_sqs_queue.background_job_queue.id,
+    execution_limit = 5
+  }]
+}
 ```
 
-**Please note that you must use the `sqs2sfThrottle` lambda as the consumer for any queue with a `maxExecutions` value** or else the execution throttling will not work correctly.
-Additionally, please allow at least 60 seconds after creation before using the queue as associated infrastructure and triggers are set up and made ready.
+#### Setup consumer for the queue
 
-#### Re-deploy your Cumulus app
+Add the `sqs2sfThrottle` Lambda as the consumer for the queue and add a Cloudwatch event rule/target to read from the queue on a scheduled basis.
+
+> **Please note**: You **must use the `sqs2sfThrottle` Lambda as the consumer for any queue with a queue execution limit** or else the execution throttling will not work correctly. Additionally, please allow at least 60 seconds after creation before using the queue while associated infrastructure and triggers are set up and made ready.
+
+`aws_sqs_queue.background_job_queue.id` refers to the [queue resource defined above](#add-a-new-queue).
+
+```hcl
+resource "aws_cloudwatch_event_rule" "background_job_queue_watcher" {
+  schedule_expression = "rate(1 minute)"
+}
+
+resource "aws_cloudwatch_event_target" "background_job_queue_watcher" {
+  rule = aws_cloudwatch_event_rule.background_job_queue_watcher.name
+  arn  = module.cumulus.sqs2sfThrottle_lambda_function_arn
+  input = jsonencode({
+    messageLimit = 500
+    queueUrl     = aws_sqs_queue.background_job_queue.id
+    timeLimit    = 60
+  })
+}
+
+resource "aws_lambda_permission" "background_job_queue_watcher" {
+  action        = "lambda:InvokeFunction"
+  function_name = module.cumulus.sqs2sfThrottle_lambda_function_arn
+  principal     = "events.amazonaws.com"
+  source_arn    = aws_cloudwatch_event_rule.background_job_queue_watcher.arn
+}
+```
 
-Once you [re-deploy your Cumulus application](../deployment/deployment-readme#update-cumulus), all of your workflow templates will be updated to the include information about your queue (the output below is partial output from an expected workflow template):
+#### Re-deploy your Cumulus application
+
+Follow the instructions to [re-deploy your Cumulus application](../deployment/upgrade.md#update-cumulus-resources). After you have re-deployed, your workflow template will be updated to the include information about the queue (the output below is partial output from an expected workflow template):
```

Comment:
Link doesn't work

Reply:
Not sure which page you were trying to go to here, was trying to find the right link

Reply:
This text in parentheses `the output below...` is not intended to be a link, just an aside